index.html

```diff
@@ -874,6 +697,250 @@
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">b</span></span><span id="MJXc-Node-127"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">i</span></span><span id="MJXc-Node-128"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span><span id="MJXc-Node-129"
+          class="mjx-mo" style=
+          "padding-left: 0.333em; padding-right: 0.333em;"><span class=
+          "mjx-char MJXc-TeX-main-R" style=
+          "padding-top: 0.077em; padding-bottom: 0.298em;">=</span></span><span id="MJXc-Node-130"
+          class="mjx-mfrac"><span class="mjx-box MJXc-stacked" style=
+          "width: 2.265em; padding: 0px 0.12em;"><span class="mjx-numerator"
+          style="width: 2.265em; top: -1.447em;"><span id="MJXc-Node-131"
+          class="mjx-msub"><span class="mjx-base" style=
+          "margin-right: -0.003em;"><span id="MJXc-Node-132" class=
+          "mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em; padding-right: 0.003em;">d</span></span></span><span class="mjx-sub"
+          style=
+          "font-size: 70.7%; vertical-align: -0.219em; padding-right: 0.071em;"><span id="MJXc-Node-133"
+          class="mjx-mrow" style=""><span id="MJXc-Node-134" class=
+          "mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+          "padding-top: 0.372em; padding-bottom: 0.372em;">16</span></span><span id="MJXc-Node-135"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">b</span></span><span id="MJXc-Node-136"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">i</span></span><span id="MJXc-Node-137"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span></span><span class="mjx-denominator"
+          style="width: 2.265em; bottom: -0.778em;"><span id="MJXc-Node-138"
+          class="mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+          "padding-top: 0.372em; padding-bottom: 0.372em;">255</span></span></span><span style="border-bottom: 1.3px solid; top: -0.296em; width: 2.265em;"
+          class="mjx-line"></span></span><span style=
+          "height: 2.225em; vertical-align: -0.778em;" class=
+          "mjx-vsize"></span></span></span></span></span></span></span>
+        </p>
+        <div class="note">
+          It is an implementation detail how the frames are rendered to the
+          screen. For example, the implementation may use a grayscale or red
+          channel representation.
```

Comment:
@bbernhar below are some quick thoughts. @ds-hwang might have some other benefits in mind.

* A single channel (red) would make it rather easy to read "depth pixels" from an `ImageData` object using the video+canvas pipeline:

```
canvas.drawImage(depthVideo, 0, 0, w, h);
var imageData = canvas.getImageData(0, 0, w, h);
var data = imageData.data; // type of Uint8ClampedArray

for (var i = 0; i < data.length; i += 4) {
  var depth = data[i];     // red channel repurposed as the depth pixel value
  // green, blue, and alpha channels are zero
  // var green = data[i + 1];
  // var blue  = data[i + 2];
  // var alpha = data[i + 3];
}
```
* Can reuse [Canvas Pixel ArrayBuffer](https://html.spec.whatwg.org/#canvas-pixel-arraybuffer) as is, just repurpose red as depth. All the existing APIs that deal with Canvas Pixel ArrayBuffer will just work.

* Can improve precision for video+canvas pipeline in the future: [`ImageData`](https://html.spec.whatwg.org/#imagedata)`.data` is of type `Uint8ClampedArray`, could be extended to support other [buffer source types](https://heycam.github.io/webidl/#idl-buffer-source-types), e.g. `Uint16Array`. Currently the video+canvas pipeline is disadvantaged over WebGL, since it is restricted to 8-bit precision per depth pixel.

Alternatively, we could use multiple channels, and define our own packing format (similarly to how it is defined for  `WebGLRenderingContext`, i.e. "the R component represents the lower 8 bit value of 16 bit depth value, the G component represents the upper 8 bit value of 16 bit depth value and the value in B component is not defined."). This would mean web developers need to do unpacking in JavaScript (vs in a shader), probably too slow (I have no evidence through).

The note ("It is an implementation detail how the frames are rendered to the screen. For example, the implementation may use a grayscale or red channel representation.") suggests implementations could render the depth map as they see fit when a depth-only stream is assigned as a media provider object regardless of the underlying `ImageData` representation. Consider:

```
var depthVideo = document.querySelector('#video');
depthVideo.srcObject = depthStream;
video.play();
```

Now that I wrote this comment, it appeared to me we'll still need to update this PR:

* Find a better name for the *convert the depth map value to grayscale" algorithm since it is not really grayscale. Suggestions?
* Note the red component of the Canvas Pixel ArrayBuffer is repurposed for depth.
* Update the recovery steps. Now they assume d<sub>16bit</sub>, but in the canvas+video pipeline we have d<sub>8bit</sub>.
* Nice to have (can be done after this lands): define the multiple sinks we have (<video>, WebGL 1 and 2) into their own section for clarity.